gandalf.js

```diff
@@ -1,24 +1,26 @@
-/**
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.